openstates/wy/bills.py

```diff
@@ -262,6 +267,11 @@ def scrape_vote(self, bill, vote_json, session):
 
         yield v
 
+    def clean_voter_name(self, name):
```

Comment:
let's use .strip() for cases like this as it'll work even if extra whitespace is added

Reply:
@jamesturk Just pushed a fix.